development/tsdb-blocks-storage-s3/config/mimir.yaml

```diff
@@ -3,6 +3,12 @@ multitenancy_enabled: false
 distributor:
   pool:
     health_check_ingesters: true
+  ring:
+    instance_addr: 127.0.0.1
+    kvstore:
+      store: consul
+      consul:
+        host: consul:8500
```

Comment:
We now need to explicitly set consul, otherwise this ring defaults to `memberlist`.